scribus/scribusdoc.cpp

```diff
@@ -6068,18 +6068,21 @@ void ScribusDoc::reformPages(bool moveObjects)
 	int rowcounter = 0;
 	double maxXPos = 0.0;
 	double maxYPos = 0.0;
-	double currentXPos = m_docPrefsData.displayPrefs.scratch.left();
-	double currentYPos = m_docPrefsData.displayPrefs.scratch.top();
 	double lastYPos = Pages->at(0)->initialHeight();
-	currentXPos += (Pages->at(0)->initialWidth() + m_docPrefsData.displayPrefs.pageGapHorizontal) * counter;
-	int rtlBinding = m_docPrefsData.docSetupPrefs.docBindingDirection;
-	if (rtlBinding == 1) {
-		counter = !counter;
-		currentXPos = m_docPrefsData.displayPrefs.scratch.left() + m_docPrefsData.docSetupPrefs.pageWidth;
-		currentXPos -= (m_docPrefsData.docSetupPrefs.pageWidth + m_docPrefsData.displayPrefs.pageGapHorizontal) *
-				counter;
+
+	bool rtlBinding = m_docPrefsData.docSetupPrefs.bindingDirection == 1;
+
+	double currentXPos = m_docPrefsData.displayPrefs.scratch.left();
+	if (rtlBinding)
+	{
+		counter = counter == 1 ? 0 : 1;
+		currentXPos +=  m_docPrefsData.docSetupPrefs.pageWidth
```

Comment:
this makes things difficult to read but shorter :+1:

Reply:
I don't like to use "not" for integers.
I prefer to only use it for for booleans.